src/utils/filings.ts

```diff
@@ -10,13 +10,16 @@ export const isFilingType =
 
 export const isStaffFiling = (filing: ApiResponseFilingI) => {
   return isFilingType(filing, FilingTypes.ADMIN_FREEZE) ||
-    isFilingType(filing, FilingTypes.COURT_ORDER) ||
     isFilingType(filing, undefined, FilingSubTypeE.DISSOLUTION_ADMINISTRATIVE) ||
     isFilingType(filing, FilingTypes.PUT_BACK_ON) ||
     isFilingType(filing, FilingTypes.REGISTRARS_NOTATION) ||
     isFilingType(filing, FilingTypes.REGISTRARS_ORDER)
 }
 
+export const isCourtOrderOrNotStaffFiling = (filing: ApiResponseFilingI) => {
+  return isFilingType(filing, FilingTypes.COURT_ORDER)
+}
```

Comment:
Where is this used?
How does this return True when it's not a staff filing?
Did you check everywhere that isStaffFiling is used and confirm that it doesn't affect court order? (In other words, whether a court order filing _should_ return True in some places?)